quarkus/tests/integration/src/test/java/org/keycloak/it/cli/StartCommandTest.java

```diff
@@ -39,13 +40,15 @@ void failNoTls(LaunchResult result) {
 
     @Test
     @Launch({ "--profile=dev", "start" })
+    @WithLegacyStoreOnly
     void failUsingDevProfile(LaunchResult result) {
         assertTrue(result.getErrorOutput().contains("ERROR: You can not 'start' the server in development mode. Please re-build the server first, using 'kc.sh build' for the default production mode."),
                 () -> "The Output:\n" + result.getErrorOutput() + "doesn't contains the expected string.");
     }
 
     @Test
     @Launch({ "-v", "start", "--http-enabled=true", "--hostname-strict=false" })
+    @WithLegacyStoreOnly
```

Comment:
Why we can't make it simpler and just annotate the class instead of individual methods?